layout/PrimaryNav/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+import React from 'react'
+import classnames from 'classnames'
+
+export default React.createClass({
+  displayName: 'PrimaryNav',
+
+  propTypes: {
+    reduce: React.PropTypes.bool
+  },
+
+  render: function() {
+    var className = classnames({
+      'Navigation--reduce': this.props.reduce
+    }, 'Navigation');
```

Comment:
Is there a reason the class names for this component differ from the component name? Not a huge deal but I found it confusing for a moment